lib/release_manager/release.rb

```diff
@@ -41,7 +46,9 @@ def bump
       return
     end
     raise TagExists.new("Tag v#{version} already exists") if puppet_module.tag_exists?("v#{next_version}", options[:remote])
-    version = puppet_module.bump_patch_version unless options[:bump]
+    if puppet_module.respond_to?("bump_#{level}_version")
```

Comment:
in order to check if a method exists you must convert it to a symbol.  This is the case for respond_to? and send()

`"bump_#{options[:level]}_version".to_sym`

Reply:
Not sure I understand.  If level is a property containing the string value  "patch".  Then I would expect "bump_#{level}_version to return the string 'bump_patch_version'

Reply:
@mjbain151 See this doc [respond_to](https://ruby-doc.org/core-2.4.2/Object.html#method-i-respond_to-3F). Also check out [this nice article on ruby symbols](http://www.troubleshooters.com/codecorn/ruby/symbols.htm#_How_are_symbols_implemented)